style.css

```diff
@@ -1 +1,156 @@
-/*  Add your styling here */
+/*  DEFAULT */
+:root {
+  --main-color: #faf1da;
+  --font-color: #5f4501;
+  --third-color: #9c6644;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+header,
+footer {
+  background-color: var(--main-color);
+  padding: 1rem;
+  color: var(--font-color);
+}
+
+.banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 90px;
+  height: 90px;
+}
+
+.header-shop-name {
+  font-family: "Lobster", cursive;
+  font-size: 1.3rem;
+  padding: 0.8rem;
+}
+
+h3.header-text {
+  width: 250px;
+  position: relative;
+  left: 8vw;
+  font-size: 1rem;
+  bottom: 10%;
+  padding: 2rem 1rem 1rem 1rem;
+  margin: 0 auto;
+  text-align: right;
+}
+
+/* Main section */
+
+main {
+  padding: 1rem;
+  text-align: center;
+  color: var(--font-color);
+}
+
+.hero-text {
+  padding: 1rem;
+  text-align: left;
+}
+
+.hero-text h1 {
+  font-family: "Lobster", cursive;
+  font-size: 1.8rem;
+  font-weight: 300;
+  padding: 1rem 0;
+}
+
+.hero-text p {
+  line-height: 1.5rem;
+}
+
+.hero-img {
+  width: 100%;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem auto;
+}
+
+.cards-container {
+  display: flex;
+  width: 100%;
+}
+
+.cards-container img {
+  width: 50%;
+  padding: 0.5rem;
+}
+
+/* Footer */
+
+footer {
+  text-align: center;
+}
+
+footer h4 {
+  text-transform: uppercase;
+}
+
+ul.social {
+  display: flex;
+  justify-content: space-around;
+  padding: 0;
+}
+
+.social img {
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+}
+
+.social li {
+  list-style: none;
+  padding: 1.5rem 0;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 718px) {
```

Comment:
I only see one media query. Remember, you were tasked with creating a page with 3 variations - small (ie. mobile), medium (ie. tablet), large (ie. desktop)

Reply:
> I only see one media query. Remember, you were tasked with creating a page with 3 variations - small (ie. mobile), medium (ie. tablet), large (ie. desktop)

Hi Luke
Thanks for and all the comments and for reviewing my work! 
I am still working on it and adding everything that is required :)